src/pages/creators/index.tsx

```diff
@@ -0,0 +1,282 @@
+      <Head>
+        <title>{marketplace.name + ' Creators'}</title>
+        <link rel="icon" href={marketplace.logoUrl} />
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content={marketplace.name} />
+        <meta property="og:title" content={marketplace.name + ' Creators'} />
+        <meta property="og:image" content={marketplace.bannerUrl} />
+        <meta property="og:description" content={marketplace.description} />
+      </Head>
+      <div className="relative w-full mb-14">
+        <div className="absolute flex items-center w-full justify-between top-[28px]">
+          <Link to="/">
+            <img
+              src={marketplace.logoUrl}
+              alt={marketplace.name}
+              className="ml-6 border-4 object-cover border-gray-900 bg-gray-900 rounded-full w-16 h-16"
+            />
+          </Link>
+          <div className="absolute flex items-center justify-end right-6">
+            {equals(
+              publicKey?.toBase58(),
+              marketplace.auctionHouse.authority
+            ) && (
+              <Link
+                to="/admin/marketplace/edit"
+                className="text-sm cursor-pointer mr-6 hover:underline "
+              >
+                Admin Dashboard
+              </Link>
+            )}
+            <Link
+              to="/creators"
+              className="text-sm cursor-pointer mr-6 underline "
+            >
+              Creators
+            </Link>
+            <WalletPortal />
+          </div>
+        </div>
+      </div>
+      <div className="w-full max-w-[1800px] px-14">
+        <div className="mt-20 mb-20">
+          <h2>{marketplace.name + ' Creators'}</h2>
+        </div>
+
+        <div className="grid grid-cols-2 gap-8 md:gap-10 lg:gap-14 mb-20">
+          {loading ? (
+            <>
+              <div className="hover:translate-sale-1.5">
+                <div className="flex flex-grid mb-2">
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                </div>
+                <div className="bg-gray-800 h-6 w-24 block" />
+              </div>
+              <div>
+                <div className="flex flex-grid mb-2">
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                  <div className="bg-gray-800 w-1/3 aspect-square" />
+                </div>
+                <div className="bg-gray-800 h-6 w-24 block" />
+              </div>
+            </>
+          ) : (
+            creatorsQuery.data?.marketplace.creators.map((creator) => {
+              return (
+                <Link
+                  className="transition-transform hover:scale-[1.02]"
+                  key={creator.storeConfigAddress}
```

Comment:
```suggestion
                  key={creator.creatorAddress}
```

storeConfigAddress is the same for all the creators.